From 238a29d97f590a026c0c22a0ea495f6073d349a8 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 3 Jan 2018 16:18:39 +0100
Subject: [PATCH 1/1] configs: x86: allow to override CONFIG_BOOTCOMMAND

Allow to override CONFIG_BOOTCOMMAND in .config.

The default command line
"ext2load scsi 0:3 01000000 /boot/vmlinuz; zboot 01000000"
is specific to Chromebooks. Let's move it to x86-chromebook.h
as suggested by Simon.

Cc: Simon Glass <sjg@chromium.org>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
v2
	Move default to x86-chromebook.h
	Remove undef in theadorable-x86-common.h
---
 include/configs/theadorable-x86-common.h | 1 -
 include/configs/x86-chromebook.h         | 6 ++++++
 include/configs/x86-common.h             | 3 ---
 3 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/include/configs/theadorable-x86-common.h b/include/configs/theadorable-x86-common.h
index c20803c535..6ddc892c4f 100644
--- a/include/configs/theadorable-x86-common.h
+++ b/include/configs/theadorable-x86-common.h
@@ -34,7 +34,6 @@
 #define CONFIG_ENV_OFFSET_REDUND	\
 	(CONFIG_ENV_OFFSET + CONFIG_ENV_SIZE)
 
-#undef CONFIG_BOOTCOMMAND
 #undef CONFIG_EXTRA_ENV_SETTINGS
 
 #define CONFIG_EXTRA_ENV_SETTINGS				\
diff --git a/include/configs/x86-chromebook.h b/include/configs/x86-chromebook.h
index 27ba9ee229..ea4dd29747 100644
--- a/include/configs/x86-chromebook.h
+++ b/include/configs/x86-chromebook.h
@@ -40,4 +40,10 @@
 					"stdout=vidconsole,serial\0" \
 					"stderr=vidconsole,serial\0"
 
+/* Command line configuration */
+#ifndef CONFIG_BOOTCOMMAND
+#define CONFIG_BOOTCOMMAND \
+	"ext2load scsi 0:3 01000000 /boot/vmlinuz; zboot 01000000"
+#endif
+
 #endif
diff --git a/include/configs/x86-common.h b/include/configs/x86-common.h
index 064c546403..020cac1fad 100644
--- a/include/configs/x86-common.h
+++ b/include/configs/x86-common.h
@@ -64,9 +64,6 @@
  * Command line configuration.
  */
 
-#define CONFIG_BOOTCOMMAND	\
-	"ext2load scsi 0:3 01000000 /boot/vmlinuz; zboot 01000000"
-
 #if defined(CONFIG_CMD_KGDB)
 #define CONFIG_KGDB_BAUDRATE			115200
 #endif
-- 
2.14.2

